.github/workflows/_linux-benchmark-h100.yml

```diff
@@ -27,8 +35,11 @@ jobs:
     env:
```

Comment:
I think you need to set `id-token` here too:

```
permissions:
  id-token: write
  contents: read
```

Reply:
Thanks! Yes it fixes the problem.